website/docs/Use-Cases/agent_chat.md

```diff
@@ -69,12 +69,16 @@ On the one hand, one can achieve fully autonomous conversations after an initial
+AutoGen, by integrating conversation-driven control utilizing both programming and natural language, inherently supports dynamic conversations. This dynamic nature allows the agent topology to adapt based on the actual conversation flow under varying input problem scenarios. Conversely, static conversations adhere to a predefined topology. Dynamic conversations are particularly beneficial in complex settings where interaction patterns cannot be predetermined.
 
-- Registered auto-reply. With the pluggable auto-reply function, one can choose to invoke conversations with other agents depending on the content of the current message and context. A working system demonstrating this type of dynamic conversation can be found in this code example, demonstrating a [dynamic group chat](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat.ipynb). In the system, we register an auto-reply function in the group chat manager, which lets LLM decide who the next speaker will be in a group chat setting.
+1. Dynamic Group Chat
+The system enables dynamic group chat through a registered auto-reply function. This function allows initiating conversations with other agents based on the current message and context. A practical example is demonstrated in [dynamic group chat](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat.ipynb) which illustrates the management of dynamic group chats and the decision process for the next speaker.
 
-- LLM-based function call. In this approach, LLM decides whether or not to call a particular function depending on the conversation status in each inference call.
-  By messaging additional agents in the called functions, the LLM can drive dynamic multi-agent conversation. A working system showcasing this type of dynamic conversation can be found in the [multi-user math problem solving scenario](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_two_users.ipynb), where a student assistant would automatically resort to an expert using function calls.
+2. Graph-Based Group Chat
+In this approach, a directed graph is fed into [GraphGroupChat](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_two_users.ipynb), an inherited class of `GroupChat`. AutoGen then performs several functions: (1) It seeks a speaker suggestion from the previous speaker, thereby reducing API calls. (2) In the absence of a suggestion, it utilizes the LLM API to nominate the next speaker amongst the eligible speakers. (3) Lastly, it randomly selects from eligible speakers, defined as successor nodes (agents) from the prior speaker (agent). Note that for simpler use cases where all nodes are interconnected, `GroupChat` suffices. However, `GraphGroupChat` is useful for more controlled scenarios requiring specific conversation paths.
```

Comment:
I want to keep "registered auto-reply" as the original bullet point. But making these two ways of group chats as two examples. I also don't want to give the impression that group chats are the only multi-agent working patterns.
cc @qingyun-wu @ekzhu @BeibinLi

Reply:
Hi @sonichi, thank you for the specific ask. I have restored bullet point 1 to the original wording, as requested. For Point 2 as well, I have removed the last sentence so that it doesn't give the impressions that `GroupChat` is the only multi-agent working pattern.